pages/index.tsx

```diff
@@ -2,11 +2,18 @@ import { GetStaticProps } from 'next';
 import { handleImages } from 'src/utils/imageHandler';
 import { Employee, massageEmployee } from 'src/employees';
 import { getEmployeesUrl } from 'src/utils/api/getEmployees';
+import { CaseJSON } from '../src/case/Case';
```

Comment:
In `tsconfig.json` I've added path aliasing as an experiment to seeing if it is better to use absolute pathing instead of relative pathing with `../../` etc. Not as problematic here, but it's nice to be consistent.

Reply:
okey nice :D changing now^^